### DISCUSSION TITLE
Making springs

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#131] [Q&A] [answered]
Presumably I could use my ELS (which im stupidly excited about getting) for making custom springs.

Ive seen others using their lathes for this but its very manual.

It would be great if I could set a custom pitch so that I could make all manner of springs. Is this possible? For example one of the springs I want to make is 10.5mm between coils, but of course thats not a standard thread.

**Comment by kachurovskiy** [comment, marked as answer]
Hi @Festivejelly, yes, just set the pitch to `10.5mm` [using the numpad](https://github.com/kachurovskiy/nanoels/tree/main/h4#setting-pitch) and use the [GEARBOX mode](https://github.com/kachurovskiy/nanoels/tree/main/h4#gearbox-mode--electronic-lead-screw).

**Reply by Festivejelly** [reply]
Fantastic, absolute game changer for my little business. Thanks.
